Libraries/BarotraumaLibs/BarotraumaCore/Utils/MathUtils.cs

```diff
@@ -10,7 +10,7 @@ namespace Barotrauma
     [Flags]
     public enum Alignment
     {
-        CenterX = 1, Left = 2, Right = 4, CenterY = 8, Top = 16, Bottom = 32,
+        Left = 1, CenterX = 2, Right = 4, Top = 8, CenterY = 16, Bottom = 32,
```

Comment:
Why was this changed? I think it could have some unintended side-effects, such as things previously defaulting to CenterX defaulting to Left, which could cause for example UI layout issues in tons of places.

Reply:
I believe I changed this one so the values would appear "in order" (left -> right, top -> bottom), although in hindsight I see this wasn't very necessary and can be reverted.